### PR TITLE
Code Review: Revise plugin documentation (#10947)

### PR DESCRIPTION
**Pull request by samdeane**
Code review for Revise plugin documentation (#10947):

> It could do with re-wording and generally updating a bit.

Connect to BohemianCoding/Sketch#10947.

### CODE REVIEW

File: _introduction/locations.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+~/Library/Application Support/com.bohemiancoding.sketch3/Plugins
+```
+
+*(~ here is shorthand for your home folder, e.g. `/Users/joe`)*
+
+You can access this Plugins folder easily by creating a Plugin with this code:
+
+```
+NSWorkspace.sharedWorkspace().openURL(MSPluginManager.mainPluginsFolderURL())
+```
```
Comment (bomberstudios) [inline _introduction/locations.md]: Maybe worth adding:

> …or holding Alt while opening the Plugins menu and then choosing 'Reveal Plugins Folder'.

File: _introduction/basics.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+order: 100
+---
+
+## What is a Plugin?
+
+On disk, a Plugin is just a folder arranged in a [standard layout](/introduction/plugin-bundles/).
+
+It contains one or more scripts. Each script defines one or more commands which extend Sketch in some way. It can also contain any other optional resources (such as images) that the commands make use of to do whatever they do.
+
+Other files in the Plugin folder include the actual scripts which implement the commands, and
```
Comment (bomberstudios) [inline _introduction/basics.md]: and… _what_? : )

(Actually, this looks like a leftover from the previous content, and we can probably get rid of the whole sentence)

Comment (samdeane) [inline _introduction/basics.md]: erm, yeah - I think that was an editing error on my part.

File: _introduction/locations.md (hunk 1 above)
Comment (samdeane) [inline _introduction/locations.md]: actually I've just removed the plugin code and mentioned the menu command instead. The plugin code was nicely self-referential, but definitely not the best way to reveal the folder now that we have the menu.

Comment (bomberstudios) [inline _introduction/locations.md]: 👍

**Comment by samdeane** [comment]
Thanks. Can you approve the main Sketch PR too - I had to make one so that GH allowed me to merge it in :)

https://github.com/BohemianCoding/Sketch/pull/10959

**Comment by bomberstudios** [comment]
on my way!

**Comment by bomberstudios** [comment]
ready : )

**Pull request merged**